source/channels/find-channels.rst

```diff
@@ -24,7 +24,8 @@ Find channels to join
 To see all available public channels you can join that you're not already a member of, select the **+** icon at the top of the channel sidebar, then select **Browse Channels**. Search for channels by name or scroll through the list. Select **Join** next to any channel to become a member of that channel.
 
 .. tip:: 
-    - Want to see all of the channels you're already a member of? Select **Find Channel** in the channel sidebar to see all of the channels you're currently a member of across all of your teams, including public and private channels, as well as all direct and group messages.
+  
+  Want to see all of the channels you're already a member of? Select **Find Channel** in the channel sidebar to see all of the channels you're currently a member of across all of your teams, including public and private channels, direct and group messages, channels with unread messages, and threads.
```

Comment:
This might be okay, but I was taught to not end a sentence in a preposition. 

`Want to see all of the channels you're already a member of?`

Reply:
You were taught well, @jfrerich :) As English has evolved over generations, these types of rules have evolved as well. Today, with the introduction of more informal communication styles, sentences can actually become more convoluted and harder to read by steadfastly avoiding prepositions at the end of sentences.

Apparently, this rule comes for 18th Century Latin! More here if you're interested: https://crosstalk.cell.com/blog/why-its-ok-and-sometimes-better-to-end-a-sentence-with-a-preposition